### ISSUE TITLE
Extracting features

### ISSUE THREAD

**Issue by shubhamagarwal92** [yuleiniu/rva#2]
Under Extracting Features (Optional), you have this step:

Prepare the MSCOCO and Flickr images.

- Where do I find the Flickr images? Could you remind me if that is the valid and test set? 
- MSCOCO images: did you download the train images from [here](http://images.cocodataset.org/zips/train2014.zip)? These are raw images, right? You didnt take any keypoints/captioning? So we have to download all the train/valid/test set? 

Any preprocessing step required after that? 

Thanks for your help :)

**Comment by yuleiniu** [comment]
> Where do I find the Flickr images? Could you remind me if that is the valid and test set?

The Flickr images can be downloaded from the offical website of VisDial [https://visualdialog.org/data](url). More details can be found from [https://visualdialog.org/challenge/2019#dataset-description](url)

> MSCOCO images: did you download the train images from here? These are raw images, right? You didnt take any keypoints/captioning? So we have to download all the train/valid/test set?

According to the description of VisDial dataset, "The v1.0 training set consists of dialogs on ~120k images from COCO-trainval" (see [https://visualdialog.org/challenge/2019#dataset-description](url)). Only the raw images are used for visual feature extraction.
